src/make_android.sh

```diff
@@ -71,23 +122,33 @@ cd ..
 
 echo --- Copying content
 rsync -av --exclude-from='exclude.android' ../sugarizer/* www
+
+rm ../sugarizer/activities.json
+mv ../sugarizer/activities.bak.json ../sugarizer/activities.json
+
 cp etoys_remote.index.html www/activities/Etoys.activity/index.html
-if [ "$1" == "minsize" -o "$2" == "minsize" -o "$3" == "minsize" -o "$4" == "minsize" ]; then
+if [ $minsize == true ]; then
 	rm -rf www/activities/Abecedarium.activity/audio/en/*
 	rm -rf www/activities/Abecedarium.activity/audio/fr/*
 	rm -rf www/activities/Abecedarium.activity/audio/es/*
 	rm -rf www/activities/Abecedarium.activity/images/database/*
 	rm -rf www/activities/Scratch.activity/static/internal-assets/*
 	sed -i -e 's/class="offlinemode"//' www/activities/Scratch.activity/index.html
 fi
-if [ "$1" != "full" -a "$2" != "full" -a "$3" != "full" -a "$4" != "full" ]; then
+if [ $full == true ]; then
```

Comment:
You've done the wrong test :-) When `full` is true the script should not call minimize...

Reply:
Ah, my bad. Fixed it.